gulpfile.babel.js

```diff
@@ -43,7 +43,7 @@ gulp.task('build:test-json', ['clean:test'], () => gulp.src('src/**/*.json')
   .pipe(gulp.dest('./dist-test/'))
 );
 
-gulp.task('build:test', ['build:test-client', 'build:test-server', 'build:test-json']);
+gulp.task('build:test', ['build:test-server', 'build:test-json']);
```

Comment:
It is because I deleted unit test and on line number 83, I added 'jest' and that does client-side test already. Also, test/browser/index.js no longer exists..... good point I do not need line 37 actually.